requirements-5.2.txt

```diff
@@ -1,5 +1 @@
-# Keep this file in sync with: https://github.com/kitconcept/buildout/edit/master/requirements.txt
-setuptools==42.0.2
-zc.buildout==2.13.4
-wheel
-zpretty
```

Comment:
Are you sure it is okay to drop the `zpretty` requirement?

Reply:
You're right. It is used in the [Makefile](https://github.com/plone/plone.restapi/blob/dfb88d317460c824fa9a99abbcc9320e2e40bc2b/Makefile#L107).

I will bring it back.

Reply:
Done